src/uattributes/uattributesvalidator.rs

```diff
@@ -346,6 +373,7 @@ impl UAttributesValidator for RequestValidator {
             self.validate_source(attributes),
             self.validate_sink(attributes),
             self.validate_permission_level(attributes),
+            validate_rpc_priority(attributes),
```

Comment:
The part of my brain that loves congruity is screaming out in pain right now, but I suppose you don't actually need anything other than `atttributes` for this :)